apps/examples/e2e/tests/export-snapshots.spec.tsx

```diff
@@ -487,6 +487,50 @@ const snapshots: Snapshots = {
 			),
 		},
 	},
+	Arrows: {
```

Comment:
Hm, now I'm not sure sure if this will catch the issue. We already have a snapshot test for Fills, which does include arrows. I ran it locally and the snapshot definitely changed, but the tests didn't detect it. I guess the change was not significant enough.

Reply:
Made the arrows `xl` now which should detect the change. Will keep this test instead of updating the fills one though.